src/segment-loader.js

```diff
@@ -596,7 +598,7 @@ export default class SegmentLoader extends videojs.EventTarget {
     }
 
     // see if we need to begin loading immediately
-    let segmentInfo = this.checkBuffer_(this.buffered_(),
+    const segmentInfo = this.checkBuffer_(this.buffered_(),
```

Comment:
The arguments to this function should have another level of indentation so it lines up with `this.buffered_()`